iothub_client/samples/iothub_client_c2d_streaming_proxy_sample/iothub_client_c2d_streaming_proxy_sample.c

```diff
@@ -89,6 +89,10 @@ static time_t g_lastTrafficPrintTime;
 #define TRAFFIC_COUNTERS_PRINT_FREQ_IN_SECS 10
 #define INDEFINITE_TIME ((time_t)-1)
 
+// This is the maximum individual payload size currently supported by Azure IoT Device Streaming Gateway.
+// When sending more data, split it it chunks with sizes no bigger than the value below.
+static const size_t MAX_WEBSOCKET_PAYLOAD_SIZE_IN_BYTES = 64000; 
```

Comment:
>64000 [](start = 58, length = 5)

This is technically not 64k that would be 65536.  I don't know if this matters or not.

Reply:
Device Streaming Gateway supports payload size up to and including 65536 bytes

Reply:
Ditto. `64*1024` looks better :)

Reply:
Changed to 65536.